cfme/tests/infrastructure/test_providers.py

```diff
@@ -222,6 +223,35 @@ def test_provider_crud(provider):
     provider.wait_for_delete()
 
 
+@pytest.mark.usefixtures('has_no_infra_providers')
+@pytest.mark.tier(1)
+@test_requirements.provider_discovery
+@pytest.mark.parametrize('verify_tls', [False, True])
+@pytest.mark.uncollectif(lambda provider:
+    not provider.one_of(RHEVMProvider) and not provider.data['verify_tls'])
+def test_provider_rhv_create_delete_tls(provider, verify_tls):
+    """Tests RHV provider creation with and without TLS encryption
+
+    Metadata:
+       test_flag: crud
+    """
+    p = copy(provider)
+
+    if not verify_tls:
+        endpoints = deepcopy(p.endpoints)
+        endpoints['default'].verify_tls = False
+        endpoints['default'].ca_certs = None
+
+        p.endpoints = endpoints
+        p.name = "{}-no-tls".format(provider.name)
+
+    p.create()
+    p.validate_stats(ui=True)
+
+    p.delete(cancel=False)
```

Comment:
REQUIRED: Please use smtg like ``request.addfinalizer(p.delete_if_exists)`` (at the beginning of the test) so the provider is removed even in case of failure.